packages/taco/test/conditions/variable-operation.test.ts

```diff
@@ -49,3 +49,95 @@ describe('validates schema', () => {
     },
   );
 });
+
+describe('new operators', () => {
```

Comment:
Since these operations are already included in the `OPERATOR_FUNCTIONS` list they are already covered in the tests on L9. They are also tested as being part of the unary functions as part of the test on L34.

Therefore, if I'm understanding these tests correctly, they are just duplicates and not needed.

Reply:
There are some integration tests that use operations, you can add these new operators to those for integration testing.